web/src/components/AttributeList/AttributeList.tsx

```diff
@@ -21,11 +22,15 @@ const AttributeList: React.FC<IProps> = ({assertions, attributeList, onCreateAss
     navigator.clipboard.writeText(value);
     setIsCopied(true);
   };
+  const {isOpen, currentStep} = useGuidedTour(GuidedTours.Trace);
```

Comment:
here @xoscar
it works because TracePage always render the AttributeList component